packages/spacecat-shared-aws-api-client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@adobe/spacecat-shared-aws-api-client",
+  "version": "1.0.0",
+  "description": "Shared modules of the Spacecat Services - AWS API Client",
+  "type": "module",
+  "main": "src/index.js",
+  "types": "src/index.d.ts",
+  "scripts": {
+    "test": "c8 mocha",
+    "lint": "eslint .",
+    "clean": "rm -rf package-lock.json node_modules"
+  },
+  "mocha": {
+    "reporter": "mocha-multi-reporters",
+    "reporter-options": "configFile=.mocha-multi.json",
+    "spec": "test/*.test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spacecat-shared.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/adobe/spacecat-shared/issues"
+  },
+  "homepage": "https://github.com/adobe/spacecat-shared#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@adobe/fetch": "4.1.1",
+    "@adobe/helix-shared-wrap": "2.0.0",
+    "@adobe/helix-universal": "4.4.1",
+    "@aws-sdk/client-cost-explorer": "^3.496.0",
+    "aws4": "1.12.0"
```

Comment:
what's this needed for? we should include only the aws libraries actually needed.